packages/buidler-core/test/internal/buidler-evm/stack-traces/test.ts

```diff
@@ -126,17 +130,36 @@ function defineDirTests(dirPath: string) {
           ? testDefinition.description
           : "Should give the right stack trace";
 
+      const compilerOptions = {
+        solidityVersion,
+        compilerPath,
+        withOptimizations: false,
+        runs: 200
+      };
+
       const func = async function() {
-        await runTest(dirPath, testDefinition, sources);
+        await runTest(dirPath, testDefinition, sources, compilerOptions);
       };
 
-      const funcWithOptimizations = async function() {
-        await runTest(dirPath, testDefinition, sources, true);
+      const funcWithFewRuns = async function() {
+        await runTest(dirPath, testDefinition, sources, {
+          ...compilerOptions,
+          withOptimizations: true,
+          runs: 200
+        });
+      };
+
+      const funcWithManyRuns = async function() {
+        await runTest(dirPath, testDefinition, sources, {
+          ...compilerOptions,
+          withOptimizations: true,
+          runs: 10000
+        });
       };
 
       const solcVersionDoesntMatch =
         testDefinition.solc !== undefined &&
-        !semver.satisfies(getSolidityVersion(), testDefinition.solc);
+        !semver.satisfies(compilerOptions.solidityVersion, testDefinition.solc);
```

Comment:
Instead of getting the version of the installed `solc`, we receive a parameter with the solidity version that will be used.

The version and the compiler path are somewhat redundant: you should be a able to get the version from the path, and vice versa. But it's probably easier to just have that duplication.